web/views.py

```
@@ -20,7 +20,7 @@
 from django.core.paginator import Paginator
 from django.db import IntegrityError, models, transaction
 from django.db.models import Avg, Count, Q, Sum
-from django.http import FileResponse, HttpResponse, HttpResponseForbidden, JsonResponse
+from django.http import FileResponse, HttpResponse, HttpResponseForbidden, JsonResponse, Http404
```

Ruff (0.8.2)

`django.http.Http404` imported but unused

Remove unused import: `django.http.Http404`

(F401)